app/javascript/components/UserDisplay/Dashboard/Collection/index.jsx

```diff
@@ -48,8 +48,9 @@ const Collection = ({
       {(!isListView && !isEmpty) && (
         <div className="dashboard__card-grid-wrapper">
           <CardGrid>
-            {collectionCards.map((card) => (
-              <div key={`${card.id}-${new Date().getTime()}`}>
+            {collectionCards.map((card, i) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <div key={`${card.id}-${i}`}>
```

Comment:
These keys _should not_ be super unique. When I was using the Date/Time react thought it needed to update all cards that had duplicates within the grid. We don't want to rerender the grid and this prevents those cards from updating improperly.